### PR TITLE
feat!: apply the changes of vrf location in Ostracon

### PR DESCRIPTION
**Pull request by zemyblue**
### Main changes
* moved Ostracon's VRF proof block header to block.
* change ostracon block.header to tendermint block.header

## Motivation and context

Ostracon's block structure is changed. So we need to change the related codes. 
related: https://github.com/line/ostracon/pull/559

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/line/lbm-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/line/lbm-sdk/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.
- [x] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

**Comment by ulbqb** [comment]
Does this PR contain changes about voter election?

**Comment by zemyblue** [comment]
> Does this PR contain changes about voter election?

No, there isn't any changes about voter selection logic in this PR. I just applied the change https://github.com/line/ostracon/pull/559. 

I added the adding voter selection in lbm-sdk as issue #888 .

**Review by ulbqb — DISMISSED** [review]

File: x/distribution/simulation/operations_test.go [hunk 1]
```diff
@@ -244,7 +246,7 @@ func (suite *SimTestSuite) getTestingValidator(accounts []simtypes.Account, comm
 	valPubKey := account.PubKey
 	valAddr := sdk.ValAddress(account.PubKey.Address().Bytes())
 	validator, err := stakingtypes.NewValidator(valAddr, valPubKey, stakingtypes.
-		Description{})
+	Description{})
```
Comment (dudong2) [inline x/distribution/simulation/operations_test.go:249]:
```go
validator, err := stakingtypes.NewValidator(valAddr, valPubKey, stakingtypes.Description{})
```
How about removing the newline?

**Review by dudong2 — DISMISSED** [review]

File: x/distribution/simulation/operations_test.go (hunk 1 above)
Comment (zemyblue) [reply x/distribution/simulation/operations_test.go:249]: This is not my changes, but it seems strange. ^^;
I'll fix it but, please approve again.

**Review by dudong2 — APPROVED** [review]

**Review by 0Tech — APPROVED** [review]

**Pull request merged**